docs/using-the-jdbc-driver/using-plugins/UsingTheSimpleReadWriteSplittingPlugin.md

```diff
@@ -0,0 +1,98 @@
+| `srwWriteEndpoint`           | String  |   Yes    | The endpoint to connect to when `setReadOnly(false)` is called.                                                                               | `null`        | `<cluster-name>.cluster-<XYZ>.<region>.rds.amazonaws.com`    |
+| `srwReadEndpoint`            | String  |   Yes    | The endpoint to connect to when `setReadOnly(true)` is called.                                                                                | `null`        | `<cluster-name>.cluster-ro-<XYZ>.<region>.rds.amazonaws.com` |
+| `verifyNewSrwConnections`    | Boolean |    No    | Enables role verification for new connections made by the Simple Read/Write Splitting Plugin.                                                 | `true`        | `true`, `false`                                              |
+| `verifyOpenedConnectionType` | String  |    No    | If `verifyNewSrwConnections` is set to `true`, this parameter will verify the initial opened connection to be either a writer or a reader.    | `null`        | `writer`, `reader`                                           |
+| `srwConnectRetryTimeoutMs`   | Integer |    No    | If `verifyNewSrwConnections` is set to `true`, this parameter sets the maximum allowed time in milliseconds for retrying connection attempts. | `60000`       | `60000`                                                      |
+| `srwConnectRetryIntervalMs`  | Integer |    No    | If `verifyNewSrwConnections` is set to `true`, this parameter sets the time delay in milliseconds between each retry of opening a connection. | `1000`        | `1000`                                                       |
+
+## How the Simple Read/Write Splitting Plugin Verifies Connections
+
+The property `verifyNewSrwConnections` is enabled by default. This means that when new connections are made with the Simple Read/Write Splitting Plugin, a query is sent to the new connection to verify its role. If the connection cannot be verified as having the correct role—that is, a write connection is not connected to a writer, or a read connection is not connected to a reader—the plugin will retry the connection up to the time limit of `srwConnectRetryTimeoutMs`. 
+
+The values of `srwConnectRetryTimeoutMs` and `srwConnectRetryIntervalMs` control the timing and aggressiveness of the plugin's retries.
+
+Additionally, to consistently ensure the role of connections made with the plugin, the plugin also provides role verification for the initial connection. When connecting with an RDS writer cluster or reader cluster endpoint, the plugin will retry the initial connection up to `srwConnectRetryTimeoutMs` until it has verified connection to the intended role of the endpoint.
+If it is unable to return a verified initial connection, it will log a message and continue with the normal workflow of the other plugins.
```

Comment:
Is this referring to the case when we successfully check the role but it does not align with what we expected? Or is this the case when we cannot determine the role we are connected to?